src/components/map/EventLayer.js

```diff
@@ -169,13 +169,18 @@ class EventLayer extends Layer {
                     ];
 
                     if (displayEl) {
-                        let value = dataValue.value;
-
-                        if (displayEl.optionSet) {
-                            value = displayEl.optionSet[value];
+                        const { valueType, optionSet, name } = displayEl;
+                        let { value } = dataValue;
+
+                        if (valueType === 'COORDINATE' && value) {
+                            value = JSON.parse(value)
+                                .map(v => v.toFixed(6))
```

Comment:
Similarly, this assumes that the returned value is an array of integers, which could easily break and crash the app.  It should be at minimum try/catch-guarded

Reply:
Wrapped in a try/catch